docs/pages/application-access/getting-started.mdx

```diff
@@ -23,38 +36,42 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
+   
+   <Admonition type="note" title="Application Access and DNS for Self-Hosted Clusters">
 
-<Tabs>
-<TabItem label="Docker">
-- A Docker installation, which we will use to launch Grafana in a container.
-  Alternatively, if you have another web application you'd like to protect with
-  Teleport, you can use that instead.
-</TabItem>
-<TabItem label="Kubernetes Cluster">
+   (!docs/pages/includes/dns-app-access.mdx!)
 
-(!docs/pages/includes/kubernetes-access/helm-k8s.mdx!)
+   Teleport Cloud will automatically provide a subdomain and signed TLS certificate for your application under your tenant address.
+   </Admonition>
 
-- Administrative rights to deploy within a cluster.
+- Docker or Kubernetes to launch Grafana.
 
-</TabItem>
-</Tabs>
+   This tutorial illustrates how to launch Grafana in a Docker container or Kubernetes cluster.
+   Therefore, you must have either Docker or Kubernetes installed. For more information, click **Docker** or **Kubernetes cluster**.
+   
+   <Tabs>
+   <TabItem label="Docker">
+   You must have some version of Docker installed to launch Grafana in a Docker container. 
```

Comment:
This line replaces "A Docker installation, which we will use to launch Grafana in a container.". I think the original is more succinct and less ambiguous (some version).

Reply:
Not a fan of the which clause, first person plural, or future tense. But I'm thinking of. a slightly different approach to the text here. It seemed odd to me that the Kubernetes tab mentions specific version information and the Docker tab didn't. My initial thought was to make the tabs more parallel, but yeah, that wasn't useful at all.